utils/fo-installdeps

```diff
@@ -110,13 +110,13 @@ if [ $BUILDTIME ]; then
               lenny|hardy) apt-get $YesOpt install $YesOpt postgresql-server-dev-8.3;;
               squeeze) apt-get $YesOpt install $YesOpt postgresql-server-dev-8.4;;
               wheezy) apt-get $YesOpt install $YesOpt postgresql-server-dev-9.1;;
-              sid) apt-get  $YesOpt install $YesOpt postgresql-server-dev-8.4;;
               lucid|natty) apt-get $YesOpt install postgresql-server-dev-8.4;;
               isadora) apt-get $YesOpt install postgresql-server-dev-8.4;;
               precise|oneiric|quantal|raring|saucy) apt-get $YesOpt install postgresql-server-dev-9.1;;
               trusty) apt-get $YesOpt install postgresql-server-dev-9.3;;
               jessie) apt-get $YesOpt install postgresql-server-dev-9.4;;
-              xenial) apt-get $YesOpt install postgresql-server-dev-9.5;;
+              xenial|yakkety) apt-get $YesOpt install postgresql-server-dev-9.5;;
+              stretch|buster|sid|zesty|artful) apt-get $YesOpt install postgresql-server-dev-9.6;;
```

Comment:
IMHO it is reasonable to remove 'sid' support - as a bleeding edge unstable 'rolling' distro, much like arch, users are expected to "know what they are doing" - and for the same reason arch support was deemed to be not a priority, it is probably an extremely small use-case that anyone is running a server on 'sid' - the docker image is still there for those few

but on that note the 'buster' distro is just as volatile trailing only a few weeks behind 'sid' so they should stay or go together - but the main thing is that this is not going to "break" until v9.7 comes out - which will not exactly be broken - it is just adding yet another version to the stack as you see has been done several times already - and regardless of debian, this would still be the case for the next bi-annual release of ubuntu that includes v9.7

practically speaking, ubuntu is actually the more troublesome target than debian because it is guaranteed to requires a routine update to these install files every 6 months - so to be consistent with this decision, note that dropping support for 'buster' and 'sid' and supporting only the LTS releases of debian is the equivalent to only supporting ubuntu LTS every 2 years and not supporting any of the interim versions as has been done all along